### DISCUSSION TITLE
Sydney metro mesh planning

### DISCUSSION THREAD

**Discussion by rthorntn** [meshtastic/Discussions#27] [South Asia/APAC]
Hi,

I'm in Lane Cove North (82m) and looking to put a 24/7 node up on my roof and want to work with folks nearby to get meshed, who has a 24/7 node nearby that wants to work with me?

Thanks.
Richard

**Comment by Soupbones** [comment]
Hi Richard.
There is already an extensive mesh now in Sydney, so get that node up and see how many others you connect with? We now have a few nodes up in high locations which are helping mesh the network in places like the blue mountains, SW Sydney and hopefully one going up Kurrajong Heights soon. One of the high nodes near me I know connects with a node in North Sydney. There are also a bunch of nodes now in Sydney South and the shire regions, as well as Macarthur and far west around Penrith.
